omod/src/main/webapp/portlets/patientOverview.jsp

```diff
@@ -36,6 +36,138 @@
 	</table>
 	<table id="patientActions">
 		<openmrs:extensionPoint pointId="org.openmrs.patientDashboard.patientActionsContent" type="html" parameters="patientId=${model.patient.patientId}"/> 
+		<tr class="patientActionsRow">
```

Comment:
Same here, if you can reference where this is copied back in from.

Most important thing here will be to ensure that this does not break anyone who chooses to continue not using this feature.